web/pdf_viewer.css

```diff
@@ -39,11 +44,49 @@
 
 .pdfViewer.singlePageView {
   display: inline-block;
+  margin: 0;
 }
 
 .pdfViewer.singlePageView .page {
   margin: 0;
   border: none;
+  display: block;
```

Comment:
Why was this added? Is this to counter the `display:inline-block` for `.spread .page` below?